Source/DafnyLanguageServer.Test/Synchronization/VerificationStatusTest.cs

```diff
@@ -443,15 +455,18 @@ iterator ThatIterator(x: int) yields (y: int, z: int)
     await AssertNoResolutionErrors(documentItem);
     var status = await verificationStatusReceiver.AwaitNextNotificationAsync(CancellationToken);
 
-    Assert.Equal(8, status.NamedVerifiables.Count);
-    Assert.Equal(new Range(1, 17, 1, 23), status.NamedVerifiables[0].NameRange);
-    Assert.Equal(new Range(7, 11, 7, 34), status.NamedVerifiables[1].NameRange);
-    Assert.Equal(new Range(12, 8, 12, 17), status.NamedVerifiables[2].NameRange);
-    Assert.Equal(new Range(15, 9, 15, 30), status.NamedVerifiables[3].NameRange);
-    Assert.Equal(new Range(20, 11, 20, 34), status.NamedVerifiables[4].NameRange);
-    Assert.Equal(new Range(27, 5, 27, 10), status.NamedVerifiables[5].NameRange);
-    Assert.Equal(new Range(30, 8, 30, 16), status.NamedVerifiables[6].NameRange);
-    Assert.Equal(new Range(33, 9, 33, 21), status.NamedVerifiables[7].NameRange);
+    Assert.Equal(9, status.NamedVerifiables.Count);
+    var index = 0;
+    Assert.Equal(new Range(1, 17, 1, 23), status.NamedVerifiables[index++].NameRange);
+    // TODO this entry doesn't actually have anything to verify, but it's hard to determine
+    Assert.Equal(new Range(4, 9, 4, 30), status.NamedVerifiables[index++].NameRange);
+    Assert.Equal(new Range(7, 11, 7, 34), status.NamedVerifiables[index++].NameRange);
+    Assert.Equal(new Range(12, 8, 12, 17), status.NamedVerifiables[index++].NameRange);
+    Assert.Equal(new Range(15, 9, 15, 30), status.NamedVerifiables[index++].NameRange);
+    Assert.Equal(new Range(20, 11, 20, 34), status.NamedVerifiables[index++].NameRange);
+    Assert.Equal(new Range(27, 5, 27, 10), status.NamedVerifiables[index++].NameRange);
+    Assert.Equal(new Range(30, 8, 30, 16), status.NamedVerifiables[index++].NameRange);
+    Assert.Equal(new Range(33, 9, 33, 21), status.NamedVerifiables[index].NameRange);
```

Comment:
Can be done in a follow-up PR, but I would love if you did not explicit the ranges there but obtain them from the test case instead. It would make the code easier to maintain.

Reply:
I would rather leave that for when this ranges are updated.